crates/subspace-node/src/commands/run/consensus.rs

```diff
@@ -406,9 +406,9 @@ pub(super) struct ConsensusChainOptions {
     #[clap(flatten)]
     timekeeper_options: TimekeeperOptions,
 
-    /// Sync mode
-    #[arg(long, default_value_t = ChainSyncMode::Snap)]
-    sync: ChainSyncMode,
+    /// Sync mode (default will be set to Snap)
```

Comment:
Would be nice to include documentation changes from https://github.com/autonomys/subspace/pull/2952 because it'll result in merge conflict. Especially because user can't specify `Snap`, it is `snap` or `full`

Reply:
 Ok. I'll copy the exact lines.